src/Network/Network/help/New-AzVpnSiteLink.md

```diff
@@ -45,6 +45,15 @@ PS C:\> $vpnSite = New-AzVpnSite -ResourceGroupName "testRG" -Name "testVpnSite"
 
 The above will create a resource group, Virtual WAN and a VpnSite with 1 VpnSiteLinks in West US in "testRG" resource group in Azure.
 
+### Example 2
+
+Creates an Azure VpnSiteLink object. (autogenerated)
+
+<!-- Aladdin Generated Example -->
+```powershell
+New-AzVpnSiteLink -BGPAsn <UInt32> -BGPPeeringAddress <String> -IPAddress '15.25.35.45' -LinkProviderName 'SomeTelecomProvider' -LinkSpeedInMbps '10' -Name 'testVpnSiteLink1'
```

Comment:
Not sure that I like a real public IP address being used that isn't owned by Microsoft. Is this real customer data?